src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java

```diff
@@ -39,6 +39,14 @@ public String getUrlForPath(String wikiName, String wikiPath) {
     return url;
   }
 
+  public String getUrlForPathWithWWW(String wikiName, String wikiPath) {
+    String url = getUrlForWikiWithWWW(wikiName);
+    String separator = wikiName.endsWith("wikia") || wikiName.equals("wowwiki") ? "" : "wiki/";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "wowwiki" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1132)